buddyboss/buddyboss-platform
# PROD-829 Fixed #2346 - Searching with ampersand character is not working

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #2346 

### How to test the changes in this Pull Request:

1. Try to  create a post, forum, or a group with a title that contains ampersand like "Sample & Test"
2. Now go to the frontend and use the Global Search field
3. Input the title
4. Notice the issue

### Proof Screenshots or Video

![ss](https://user-images.githubusercontent.com/58522498/115184580-ada8c680-a110-11eb-8bdb-8cf66feae9cc.jpg)

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
